venues/ICLR.cc/2019/Conference/python/bidding_stage.py

```diff
@@ -27,13 +28,58 @@
     with open('../webfield/reviewerWebfieldBiddingEnabled.js','r') as f:
         reviewers = client.get_group(iclr19.REVIEWERS_ID)
         reviewers.web = f.read()
-        client.post_group(reviewers)
+        reviewers = client.post_group(reviewers)
 
     with open('../webfield/areaChairWebfieldBiddingEnabled.js','r') as f:
         area_chairs = client.get_group(iclr19.AREA_CHAIRS_ID)
         area_chairs.web = f.read()
-        client.post_group(area_chairs)
+        area_chairs = client.post_group(area_chairs)
 
     iclr19.add_bid.invitees = [iclr19.REVIEWERS_ID, iclr19.AREA_CHAIRS_ID]
     client.post_invitation(iclr19.add_bid)
 
+    # set up User Score notes
+
+    if not all(['~' in member for member in reviewers.members]):
+        print('WARNING: not all reviewers have been converted to profile IDs. Members without profiles will not have metadata created.')
+    valid_reviewer_ids = [r for r in reviewers.members if '~' in r]
+
+    if not all(['~' in member for member in area_chairs.members]):
+        print('WARNING: not all area chairs have been converted to profile IDs. Members without profiles will not have metadata created.')
+    valid_ac_ids = [r for r in area_chairs.members if '~' in r]
+
+    client.post_invitation(iclr19.scores_inv)
+    user_score_notes = {}
+    for paper in openreview.tools.iterget_notes(client, invitation=iclr19.BLIND_SUBMISSION_ID):
+        for user_id in valid_reviewer_ids + valid_ac_ids:
+            if user_id not in user_score_notes:
+                user_score_notes[user_id] = openreview.Note.from_json({
+                    'invitation': iclr19.SCORES_INV_ID,
+                    'readers': [user_id],
+                    'writers': ['ICLR.cc/2019/Conference'],
+                    'content': {
+                        'users': [user_id],
+                        'scores': []
+                    },
+                    'signatures': ['ICLR.cc/2019/Conference'],
+                    'forum': None,
+                    'multiReply': False
+                })
+            score_note = user_score_notes[user_id]
+
+            score_entry = {
```

Comment:
Is there a way to do this less verbose? I'm worried about having 3000 entries.

Reply:
I'm not sure - in each user's score note, we need the forum ID and score for each paper.

the most compressed way that I can think to do it is to store it as a big string, something formatted like this:

"forum=ABCxyz;score=0.76|forum=123abc;score=0.35|..."

then we could parse it in the webfield. would this be any better, though? I don't really know